docs/getting-started.md

```diff
@@ -5,209 +5,209 @@ sidebar_class_name: sidebar-item-getting-started
+  }
+}
+
+// A simple function that calls all functions in order. You probably want more error handling.
+async function run() {
+  const cacheClient = await createCacheClient();
+
+  await createCache(cacheClient);
+
+  await listCaches(cacheClient);
+
+  await writeToCache(cacheClient, CACHE_NAME, "code", "12345");
+  await readFromCache(cacheClient, CACHE_NAME, "code");
+}
+
+run();
+```
+
+Run code
+
+```cli
+node test.js
+```
+
+The output should look something like this:
+
+```cli
+[2023-05-21T00:56:37.819Z] INFO (Momento: CacheClient): Creating Momento CacheClient
+[2023-05-21T00:56:37.831Z] INFO (Momento: ControlClient): Creating cache: demo-cache
+Cache created.
+Found caches:
+ - demo-cache
+Key stored successfully!
+Cache hit:  12345
+```
+
+   </TabItem>
+</Tabs>
 
-Momento will email your auth token to you. You can configure your local CLI to use this token by running the `momento configure` command:
-
-```
-$ momento configure
-Token: < Enter token from email here >
-Default Cache [default-cache]: < Name of cache to use on CLI by default >
-Default TTL Seconds [600]: 30 // Sets the default TTL for cache entries. For demonstration purposes we are setting this lower right now.
-[2022-03-31T15:31:25Z INFO  momento::commands::cache::cache_cli] creating cache...
-[2022-03-31T15:31:33Z INFO  momento::commands::configure::configure_cli] default cache successfully created
-```
-
-You are now up and running with Momento Cache! Let's see how to use the Momento CLI to cache data in Momento Cache.
-
-## Cache some data
-
-The `cache` command in the Momento CLI is used to interact with your Momento cache. Let's see a few of these commands in action.
-
-First, you can use the `set` subcommand to store an item in your default Momento Cache:
-
-```
-$ momento cache set --key test --value value
-[2022-03-31T15:45:17Z INFO  momento::commands::cache::cache_cli] setting key: test into cache: my-first-cache
-[2022-03-31T15:45:18Z INFO  momento::commands::cache::cache_cli] set success
-```
-
-Then, you can retrieve the key using the `get` subcommand:
-
-```
-$ momento cache get --key test
-[2022-03-31T15:45:25Z INFO  momento::commands::cache::cache_cli] getting key: test from cache: my-first-cache
-value
-```
+## Try our CLI and an SDK in your browser
 
-In configuring the cache, we set a default Time to Live (TTL) of 30 seconds. If you wait 30 seconds and try to retrieve the cache again, you will see the item has disappeared:
+You can try out Momento Cache today, right in your browser. No software installation required!
 
-```
-$ sleep 30 // wait for item to expire
-$ momento cache get --key test
-[2022-03-31T15:46:02Z INFO  momento::commands::cache::cache_cli] getting key: test from cache: my-first-cache
-[2022-03-31T15:46:03Z INFO  momento::commands::cache::cache_cli] cache miss
-```
+:::note
+Momento Cache is an API based serverless service. It does not deploy any resources in your (AWS, GCP, Azure, etm.) accounts.
```

Comment:
Maybe instead of `API Based` try `Fully Managed` ? not 100% on that just API based sounds bit off to me.

Reply:
Added some text around it based on your feedback. Let me know what you think please.